metricbeat/docs/modules/system.asciidoc

```diff
@@ -55,6 +47,16 @@ metricbeat.modules:
   period: 15m
   metricsets:
     - uptime
+    #- core
```

Comment:
We probably can remove this and the following lines from the `config.yml`. Or would you keep them in the advertise the others?

Reply:
Yes, I though it could be fine to leave them for visibility. I could also leave them in another commented-out module block.

Reply:
I have finally added another commented-out module block for these non-default metricsets.